components/eam/src/chemistry/mozart/linoz_data.F90

```diff
@@ -347,7 +350,8 @@ subroutine linoz_data_adv( pbuf2d, state )
     real(r8) :: to_mmr(pcols,pver)
 
     if( .not. has_linoz_data ) return
-
+    !!set true for linoz_v3 interpolation in interpolation
+    file%linoz_v3=.true.
```

Comment:
`has_linoz_data` is true when using either linozv2 or linozv3. Are your changes compatible with linozv2? If so, do we need to specify v3 here and in the related changes? If not, you need to revise it to be compatible.